packages/motion-start/src/components/AnimatePresence/__tests__/NestedExitOrderFixture.svelte

```
@@ -0,0 +1,31 @@
+<script lang="ts">
+import { motion } from '../../../index.js';
+import AnimatePresence from '../AnimatePresence.svelte';
+
+let { order }: { order: string[] } = $props();
```

Biome (2.5.5)

[error] This variable order is unused.

(lint/correctness/noUnusedVariables)